packages/components/ContextualMenu/src/SubmenuItem.tsx

```diff
@@ -124,14 +131,15 @@ export const SubmenuItem = compose<SubmenuItemType>({
     icon: { slotType: Icon as React.ComponentType<object> },
     content: Text,
     rightstack: { slotType: View },
-    chevron: { slotType: Icon as React.ComponentType<object> }
+    chevron: SvgXml
```

Comment:
What's the difference between `SvgXml` and `SVG`? I've seen the latter as a slot but not the former.

Reply:
It extends the Svg interface but take an xml value as its source